README.md

```diff
@@ -4,477 +4,43 @@
 
 <p align="center">
   <a href="https://github.com/signalfx/signalfx-java/actions?query=workflow%3A%22CI+build%22">
-    <img alt="Build Status" src="https://img.shields.io/github/workflow/status/signalfx/signalfx-java/CI%20build?style=for-the-badge">
+    <img alt="Build Status" src="https://github.com/signalfx/signalfx-java/actions/workflows/ci.yaml/badge.svg">
   </a>
 </p>
 
 ---
 
 # SignalFx client libraries
 
-This repository contains libraries for instrumenting Java applications and
-reporting metrics to SignalFx. You will need a SignalFx account and organization
-API token to use them. For more information on SignalFx and to create an
-account, go to [http://www.signalfx.com](http://www.signalfx.com).
+This repository contains legacy libraries for instrumenting Java applications and
+reporting metrics to Splunk Observability Cloud (formerly SignalFx).
 
-We recommend sending metrics with Java using Codahale Metrics version 3.0+. You
-can also use Yammer Metrics 2.0.x (an earlier version of Codahale Metrics). More
-information on the Codahale Metrics library can be found on the
-[Codahale Metrics website](https://dropwizard.github.io/metrics/).
+Splunk has adopted OpenTelemetry. Use the 
+[OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java) or
+the [Splunk Distribution of OpenTelemetry Java Instrumentation](https://github.com/signalfx/splunk-otel-java)
+agent to send telemetry data to Splunk. Do not integrate `signalfx-java` 
+into new services.
 
-You can also use the module `signalfx-java` to send metrics directly to SignalFx
-using protocol buffers, without using Codahale or Yammer metrics.
+To send data to Splunk, you need a Splunk Observability account and organization
```

Comment:
```suggestion
To send data to Splunk, you need a Splunk Observability Cloud account and organization
```

Reply:
I'm not going to say what I really think of this. Heh. 🙊